### DISCUSSION TITLE
Use of GPU on linux with AMD or Intel GPUs

### DISCUSSION THREAD

**Discussion by slangerosuna** [huggingface/candle#2486] [Q&A]
Hello! Given that candle only supports Metal and CUDA, I was wondering why there aren't any implementations for something that runs on AMD or Intel GPUs like OpenCL or WebGPU.

**Comment by def-roth** [comment]
That's a requested feature and is on the [plan.](https://github.com/huggingface/candle/issues/344#issuecomment-1669913893)

**Comment by Corallus-Caninus** [comment]
there is a fork of someone who got some AMD support (not webgpu but native), check the branches. I am going to be looking into this as well for my AMD instinct MI25 if you'd like correspondence.
#346
